community/ibm-strongloop-dev/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Open-source solutions for the API developer community
```

Comment:
This doesn't really describe what this is other than the fact that the solutions are OS. Something along the lines of what the chart actually installs would be an improvement.